packages/shared/test/fixtures/shared-secret-vectors.json

```diff
@@ -0,0 +1,235 @@
+{
+  "test_vectors": [
+    {
+      "id": "vector1",
+      "description": "Fixed nonce encryption with known plaintext",
+      "shared_secret": [
+        0,
```

Comment:
Is this an array of int bytes? Can this be in hex instead? Same for the nonce, and rust ciphertext? 

Or is there a reason for keeping this format?

Reply:
👍 - keeping it similar to the Rust implementation is a very fair point.

Reply:
Great!
Also the code accept both Uint8Array and hex string. It will be up to the Rust-generated json file to have the data in the convenient type.